api/v1/sp-eligibility-criteria.go

```diff
@@ -1,41 +1,80 @@
 package api
 
 import (
+	"encoding/json"
 	"net/http"
 
+	"github.com/data-preservation-programs/spade-tenant/db"
 	"github.com/labstack/echo/v4"
+	"gorm.io/gorm"
 )
 
-type EligibilityCriteria struct {
-	Clauses []EligibilityClause `json:"clauses"`
+func (a *apiV1) ConfigureSpEligibilityCriteriaRouter(e *echo.Group) {
+	g := e.Group("/sp")
+	g.POST("/eligibility-criteria", a.handleSetSpEligibilityCriteria)
+	g.GET("/eligibility-criteria", a.handleGetSpEligibilityCriteria)
 }
 
-type EligibilityClause struct {
-	Attribute string `json:"attribute"`
-	Operator  string `json:"operator"`
-
-	Value interface{} `json:"value"` // TODO: type - either []string or string
+type TenantSPEligibilityClausesResponse struct {
```

Comment:
Is there a way to not have the `gorm` annotations in this struct? IMO they should solely be in the `db/schema.go` and not in the api layer

Reply:
yeah we can do that.